docs/12.0/guides/upgrade-and-migration/migrating-from-11.1-to-12.0.md

```diff
@@ -220,3 +220,26 @@ To keep the previous (pre-12.0) behavior of a default keyboard shortcut, use the
 - [Remove a default keyboard shortcut](@/guides/accessories-and-menus/keyboard-shortcuts.md#removing-a-keyboard-shortcut)
 - [Replace a default keyboard shortcut](@/guides/accessories-and-menus/keyboard-shortcuts.md#replacing-a-keyboard-shortcut)
 - [Block a default keyboard shortcut's action](@/guides/accessories-and-menus/keyboard-shortcuts.md#blocking-a-keyboard-shortcut-s-actions)
+
+## Step 5: Avoid referring to `_wt`
+
+Handsontable 12.0.0 makes it clear that Handsontable's rendering engine (`_wt`, internally referred to as "Walkontable") is not a part of Handsontable's public API.
+
+To emphasize this, we changed the following property name:
+
+| Before        | After          |
+| ------------- | -------------- |
+| `hot.view.wt` | `hot.view._wt` |
```

Comment:
I propose to add:

In the above table, `hot` is the name of your Handsontable instance.

Reply:
Same as [here](https://github.com/handsontable/handsontable/pull/9550#discussion_r897640859)